### ISSUE TITLE
Building Packet.db?

### ISSUE THREAD

**Issue by Quinchu** [DaVarga/slingxdcc#2]
Might seem like a stupid question. 

I've added the server and channels I wanted to connect to. But when I search anything I don't get anything available. I think I need to build the packet.db? But I haven't been able to find any material on how to build it so I'm able to search.

It's working fine on Ubuntu 14.04 LTS, just having this one problem.

**Comment by sziep** [comment]
I guess you have tried to click on the "observe" icon next to the Channel? This should join the channel and record the announced packages in the Packets.db.

**Comment by Quinchu** [comment]
I've had this active for about ~3 days now and I've still yet to get anything. And I have the observe function selected in the channels / servers menu.

**Comment by Quinchu** [comment]
Could it have anything to do with the regex and what I'm looking for?

**Comment by DaVarga** [comment]
Yes, try to delete your settings.json and restart slingxdcc. This creates a new settings.json with the default regex which works in most cases.

**Comment by Quinchu** [comment]
I've deleted the settings files on both my windows test box and on my Linux test box. Using the default regex isn't working for me.

I'm using it mainly for anime. So sitting in irc.Rizon.net #intel they post packs every couple of minutes and neither of my test machines are picking them up.

**Comment by Quinchu** [comment]
Oh, and for reference, I'm running node 4.1.1 on both, Ubuntu 14.04LTS and Windows Server 2012 R2.

**Comment by DaVarga** [comment]
The messages in the channel are in an uncommon format. Adjust reg-ex won't be a solution here because the groups in the reg-ex must in the correct order.(packnumber, gets, filesize, filename).
I will implement a feature in the next version.

**Comment by Quinchu** [comment]
Oh wow, that would be great! I'm looking forward to it. And thank you for the quick response.
